### DISCUSSION TITLE
help my account was hacked

### DISCUSSION THREAD

**Discussion by plongcaomate** [community/community#126841] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

my account wad hacked the name of my account is plongskyut" but i cant see my account in github name is changed by hacker this is the message of github from my gmail account "Hello Plongskyut,

We wanted to let you know that your GitHub password has changed.

If you did not perform this action, you can recover access by entering [redacted by moderator] into the form at https://github.com/password_reset

To see this and other security events for your account, visit https://github.com/settings/security-log

If you run into problems, please contact support by visiting https://github.com/contact

Please do not reply to this email with your password. We will never ask for your password, and we strongly discourage you from sharing it with anyone.

**Comment by ExpiredWater** [comment]
Hi @plongcaomate  ,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Reply by ebndev** [reply]
Hi @ExpiredWater,

Thanks for jumping in to help other users find answers faster. Please note: You copied and posted a reply used by GitHub staff, and in doing so may give another user the wrong impression that you are an employee. I copied the text below and bolded so you can see what is confusing.

You do not have moderation privileges and are not a GitHub employee. Going forward please use caution and remove any wording that could mislead other users.

> but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.
> 
> As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Comment by ebndev** [comment]
Hi @plongcaomate,

As you may have seen, please open a ticket on our [Support page](https://support.github.com/contact) for account related questions as they are the only ones who are able to assist with your issue.

Thanks for understanding!
